python/ml4ir/applications/ranking/tests/test_aux_metrics_helper.py

```diff
@@ -0,0 +1,290 @@
+import unittest
+from unittest.mock import patch
+
+import pandas as pd
+import numpy as np
+from pandas import testing as pd_testing
+
+from ml4ir.applications.ranking.model.metrics.helpers.aux_metrics_helper import *
+
+
+class ComputeAuxMetricsTest(unittest.TestCase):
```

Comment:
@mohazahran This is the same old test `test_metrics_helper`. I've just cleaned it up, added documentation, etc. Not sure why it is showing up as a new file instead of a changed file on git though.